paasta_tools/kubernetes_tools.py

```diff
@@ -1421,15 +1423,15 @@ def get_autoscaled_instances(self) -> Optional[int]:
         try:
             if self.get_persistent_volumes():
                 return (
-                    KubeClient()
+                    KubeClient(__file__)
```

Comment:
this won't be that useful here, right? is it worth doing something fancy to get the name of the calling module?

Reply:
yeah probably not, I thought about doing some stack inspection stuff like I did here https://github.com/goauthentik/authentik/blob/master/authentik/events/models.py#L201-L203, but in a lot of cases that would just return `__main__` since we call the scripts directly

Reply:
we could copy that and inspect __file__ though right?

Reply:
but I think this is also only a band-aid until we have proper traces for paasta so probably dont have to worry about this too much

Reply:
true, yeah that would be cleaner